src/scripts/Build-FinOpsPsModule.ps1

```diff
@@ -0,0 +1,78 @@
+param
+(
+    [Parameter(Mandatory = $true)]
+    [string]
+    [ValidateScript({$_ -match '^(0|[1-9]\d*)\.(0|[1-9]\d*)\.(0|[1-9]\d*)(?:-((?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*)(?:\.(?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*))*))?(?:\+([0-9a-zA-Z-]+(?:\.[0-9a-zA-Z-]+)*))?$'})]
+    $Version,
+
+    [Parameter()]
+    [ValidateSet('alpha', 'beta')]
+    [string]
+    $PrereleaseTag
+)
+
+$rootPath = (Get-Item -Path $PSScriptRoot).Parent.Parent.FullName
+$moduleName = 'FinOpsToolkit'
+$moduleFullName = "$moduleName.psm1"
+$modulePath = Join-Path -Path $rootPath -ChildPath "src\powershell\$moduleFullName"
```

Comment:
Use forward slashes instead of backward slashes so the script works on Mac. Same below.

Reply:
Join-Path converts directory separators for you depending on the environment you're using, even if the child path has backward slashes in string. Do we want to standardize this going forward seeing as how a lot of other scripts use \ instead of /?

Reply:
PowerShell repo doesn't seem consistent either 
![image](https://github.com/microsoft/finops-toolkit/assets/26576969/decc0c61-7615-48a2-856d-d6939de41057)
I'll just use forward slashes